features.tex

```diff
@@ -65,8 +63,9 @@
     Legendre polynomial, Hermite polynomial, Laguerre polynomial, and
     spherical harmonic functions.\\
 \hline
-\end{tabular}
-\end{table}
+\caption{SymPy Features and Descriptions}
+\label{features-table}
```

Comment:
Hmm Section 3 of the docs probably explain how to change this: http://ctan.mirrors.hoobly.com/macros/latex/required/tools/longtable.pdf

Have to run to class right now, though.